tests/test_registercloudguest.py

```diff
@@ -2370,6 +2370,43 @@ def test_register_modules(mock_register_product):
     )
 
 
+@patch('cloudregister.registerutils.register_product')
+def test_register_modules_failed_credentials(mock_register_product):
+    prod_reg_type = namedtuple(
+        'prod_reg_type', ['returncode', 'output', 'error']
+    )
+
+    mock_register_product.return_value = prod_reg_type(
+        returncode=67,
+        output='',
+        error='missing system credentials try again'
```

Comment:
This is just nitpick: It would be good to use the real full error message as it came from a real call of the command for the test. I might be wrong but the above seems like an artificial error message produced to match our code

Reply:
Yes it's fabricated but we cannot track, at least I am not aware of a mechanism, when the real error message ( https://github.com/SUSE/connect-ng/pull/272/files#diff-5018489c1a47a658436f1cc583e09b0289c1c1416d0f80e77620da54c3742239R350) changes, as such I am not convinced there is an advantage of having the real message in the test.